src/test/ui/deprecation/deprecation-in-future.stderr

```diff
@@ -0,0 +1,14 @@
+error: use of item deprecated in future version 'deprecated_future': text
+  --> $DIR/deprecation-in-future.rs:17:5
+   |
+LL |     deprecated_future(); //~ ERROR use of item deprecated in future version 'deprecated_future': text
```

Comment:
This looks like `deprecated_future` is the version number. Could it show the version number as well?